README.md

```diff
@@ -34,6 +34,21 @@ Install other python packages:
 pip3 install -r requirements.txt
 ```
 
+## Docker Installation
```

Comment:
maybe `###`?

And add another sub-heading "## Plain installation" (or something similar) before "Download the model data"

Reply:
Sounds good to me, pushing up that change now.